### ISSUE TITLE
electrum server install ImportError

### ISSUE THREAD

**Issue by kuaikuaikim** [spesmilo/electrum-server#86]
I have a trouble when i install the electrum server.The electrum server install document reference(https://github.com/spesmilo/electrum-server/blob/master/HOWTO.md).everything has been done before run "electrum-server start" command to launch electrum server.but it print an error: "ImportError: /home/bitcoin/.python-eggs/plyvel-0.9-py2.7-linux-x86_64.egg-tmp/plyvel/_plyvel.so: undefined symbol: _ZN6snappy13RawUncompressEPKcmP".

I have google with the keyword "_ZN6snappy13RawUncompressEPKcmP". i found a reference in this page("https://github.com/wbolster/plyvel/issues/5"). I do as this page said, but finally still failed. my leveldb is 1.9, i install the liblevedb1,libleveldb-dev,plyvel. can anybody help me to solve this big problem.thanks a lot!

**Comment by joshafest** [comment]
have you installed leveldb?

**Comment by joshafest** [comment]
look bitcoind debug ...maybe there were headers only not blocks.....
you need to resync bitcoind.

**Comment by jangorecki** [comment]
Yes the installation steps are broken, the easiest way to reproduce is just start clean Ubuntu 16.04 and go step by step with HOWTO doc. I was able to fix the issue by installing various python dev related stuff, cannot tell with exactly was the case. This is quite sad the issue was not resolved since 2014. There could be a CI setup to start electrum-server in regtest mode.
